pkg/reconciler/v1alpha1/serverlessservice/resources/services_test.go

```diff
@@ -68,9 +68,10 @@ func TestMakeService(t *testing.T) {
 				Name:      "collie-pub",
 				Labels: map[string]string{
 					// Those should be propagated.
-					serving.RevisionLabelKey: "collie",
-					serving.RevisionUID:      "1982",
-					networking.SKSLabelKey:   "collie", // <= This one we add.
+					serving.RevisionLabelKey:  "collie",
+					serving.RevisionUID:       "1982",
+					networking.SKSLabelKey:    "collie",
+					networking.ServiceTypeKey: "Public",
```

Comment:
Use constant for value?

Reply:
I think we use  strings in many UTs, but if you want I can switch to constant.